frontend/packages/console-dynamic-plugin-sdk/src/app/features.ts

```diff
@@ -0,0 +1,15 @@
+import { action } from 'typesafe-actions';
+
+export enum ActionType {
+  SetFlag = 'setFlag',
+  ClearSSARFlags = 'clearSSARFlags',
+}
+
+export const setFlag = (flag: string, value: boolean) =>
+  action(ActionType.SetFlag, { flag, value });
+
+export const featureReducerName = 'FLAGS';
+
+export type FeatureSubStore<T> = {
+  [featureReducerName]: T;
+};
```

Comment:
Why not just a simple
```suggestion
export type FeatureSubStore = {
  FLAGS: ImmutableMap<string, boolean>;
};
```

 ?

I think that using `featureReducerName` constat is a pre-TS thing, we do not really need it anymore.
Also there is no reason (AFAIK) to have generics here

Reply:
ack.